File: BlackjackAPI/Models/Game.cs
using System;
namespace BlackjackAPI.Models
{
    public class Game
    {
        //getset
        public int PlayerId { get; set; }
        public string? GameStatus { get; set; }
        public Card[] Player { get; set; }
        public Card[] Dealer { get; set; }

        public Card[] Deck { get; set; }
    }
}

File: Websites/app/Models/BlackjackModels/Game.php
<?php

namespace App\Models\BlackjackModels;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

use App\Models\BlackjackModels\Card;

class Game extends Model
{
    use HasFactory;
    public string $GameStatus;
    public string $Player;
    public string $Dealer;

}
